src/PdoEventStore/ClientOperations/AppendToStreamOperation.php

```diff
@@ -78,6 +78,14 @@ public function __invoke(
 SQL
             );
             $statement->execute([$streamId, $stream, 0, 0]);
+
+            $statement = $connection->prepare(<<<SQL
```

Comment:
please remove this code, it's meant to be here.

Reply:
default ACL is defined by system settings, see https://github.com/prolic/awesomeness/blob/master/src/EventStore/SystemSettings.php#L23

Reply:
Yes, this is a hack meant to be removed.

The thing is that the `LoadStreamIdOperation` requires to have a list of roles in order to work.

Otherwise, `explode` will be called with arguments `', '` and `null` [here](https://github.com/prolic/awesomeness/blob/master/src/PdoEventStore/ClientOperations/LoadStreamIdOperation.php#L91) which will cause an exception.

As I was not sure about the wanted behaviour, I've pushed this ugly hack.

Reply:
LoadStreamIdOperation needs to be fixed than and return empty array of roles

Reply:
Sorry, never tested it :)

Reply:
You mean something like [this](https://github.com/prolic/awesomeness/pull/2/files#diff-0675cec84adc75965047a58c9657c38eR90)?